packages/uswds-core/src/js/polyfills/svg4everybody.mjs

```diff
@@ -1,7 +1,4 @@
-/* eslint-disable */
-!(function (factory) {
-  module.exports = factory();
```

Comment:
Just curious - what did this do?

Reply:
This polyfill used Universal Module Definition (UMD) style when creating the pattern. This pattern was used to ensure global scope (whether in Node or browser) wasn't polluted. It was also used for improved compatibility between different patterns.

The last update was in 2017 and I believe we added some changes because it was breaking in webpack https://github.com/uswds/uswds/pull/4459.